src/composables/transactionList.ts

```diff
@@ -58,18 +61,17 @@ function updateAccountTransaction(address: Encoded.AccountAddress, transaction:
   }
 }
 
-const setTransactionsNextPage = (address: Encoded.AccountAddress, url: string) => {
+const setTransactionsNextPage = (address: string, url: string | null) => {
   transactions.value[address].nextPageUrl = url;
 };
 
 export function useTransactionList({ store }: IDefaultComposableOptions) {
-  const { aeActiveNetworkSettings } = useAeNetworkSettings();
+  const { onNetworkChange } = createNetworkWatcher();
```

Comment:
`createNetworkWatcher` should be initialized before the actual composable scope.